### PR TITLE
Added edit button to sessions

### PR DESCRIPTION
**Pull request by jankrzempek**
For now only view of the buttons - I wanted to split it into smaller parts, hope it's ok

https://trello.com/c/6LySWQdY/44-edit-button-on-session-view

### CODE REVIEW

File: AirCasting/Utils/Strings.swift [hunk 1]
```diff
@@ -34,4 +34,10 @@ struct Strings {
         static let Logged: String = "You are currently logged in as admin@admin.com"
         static let signOut: String = "Sign Out"
     }
+    
+    struct sessionShare {
```
Comment (smialko) [inline AirCasting/Utils/Strings.swift:38]: Text color to aircasting gray or dark gray :D

**Review by smialko — APPROVED** [review]
🥷🏻

File: AirCasting/SessionViews/SessionHeaderView.swift [hunk 2]
```diff
@@ -8,24 +8,34 @@
 import SwiftUI
 
 struct SessionHeaderView: View {
-    
+    func placeOrder() {}
+    func adjustOrder() {}
+    func cancelOrder() {}
```
Comment (smialko) [inline AirCasting/SessionViews/SessionHeaderView.swift:13]: I guess they're for the future but I wouldn't put them here if there's no need for that (aka you don't use empty funcs for example for buttons). 
And I don't understand - what are they going to do? The buttons say resume recording, edit, share or delete and those funcs names are different :D

File: AirCasting/SessionViews/SessionHeaderView.swift [hunk 3]
```diff
@@ -8,24 +8,34 @@
...
+            })
             nameLabelAndExpandButton
             if session.deviceType == .MIC {
                 HStack {
                     measurementsMic
                     Spacer()
-                    //This is a temporary solution for stopping mic session recording until we implement proper session edition menu
-                    if microphoneManager.session?.uuid == session.uuid, microphoneManager.isRecording && (session.status == .RECORDING || session.status == .DISCONNETCED) {
+                    // This is a temporary solution for stopping mic session recording until we implement proper session edition menu
+                    if microphoneManager.session?.uuid == session.uuid, microphoneManager.isRecording, session.status == .RECORDING || session.status == .DISCONNETCED {
```
Comment (smialko) [inline AirCasting/SessionViews/SessionHeaderView.swift:38]: nice catch 😎

File: AirCasting/SessionViews/SessionHeaderView.swift [hunk 4]
```diff
@@ -97,7 +106,37 @@ private extension SessionHeaderView {
                 .foregroundColor(.accentColor)
         })
     }
-        
+    
+    var popUpMenu: some View {
```
Comment (smialko) [inline AirCasting/SessionViews/SessionHeaderView.swift:110]: Not sure about this name but I tbh don't have a better idea 🙈

File: AirCasting/SessionViews/SessionHeaderView.swift [hunk 5]
```diff
@@ -111,7 +150,7 @@ struct SessionHeader_Previews: PreviewProvider {
                           session: SessionEntity.mock,
                           threshold: .mock,
                           selectedStream: .constant(nil))
-        .environmentObject(MicrophoneManager(measurementStreamStorage: PreviewMeasurementStreamStorage()))
+            .environmentObject(MicrophoneManager(measurementStreamStorage: PreviewMeasurementStreamStorage()))
```
Comment (smialko) [inline AirCasting/SessionViews/SessionHeaderView.swift:153]: Nice catch vol 2 🤓

File: AirCasting/SessionViews/ShareViewModal.swift [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+                title
+                description
+            }
+            checkBox
+            shareButton
+            descriptionMail
+            createTextfield(placeholder: "Email", binding: $email)
+                .padding(.vertical)
+            oKButton
+            cancelButton
```
Comment (smialko) [inline AirCasting/SessionViews/ShareViewModal.swift:23]: Hmm maybe you could add a VStack for ok and cancel buttons and make smaller spacing there, like 5 or 8? Just for those two.

File: AirCasting/SessionViews/ShareViewModal.swift [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+        }
+        .padding()
+    }
+    
+    private var title: some View {
+        Text(Strings.sessionShare.title)
+            .font(.title)
```
Comment (smialko) [inline AirCasting/SessionViews/ShareViewModal.swift:30]: Are you sure it's the right font? :) 
We don't use system fonts. 
I see two possibilities here --> either use our Font.muli/Font.moderat or create an extension for this and set the right font for .title, .subheadline, etc :) 
The second solution has more advantages - it supports some dynamic text scaling. 
But it's your call :)

File: AirCasting/SessionViews/ShareViewModal.swift [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+            .foregroundColor(.darkBlue)
+            .bold()
+    }
+    
+    private var description: some View {
+        Text(Strings.sessionShare.description)
+            .font(.subheadline)
```
Comment (smialko) [inline AirCasting/SessionViews/ShareViewModal.swift:37]: same as above

**Review by smialko — COMMENTED** [review]
🥷🏻

File: AirCasting/SessionViews/ShareViewModal.swift (hunk 7 above)
Comment (jankrzempek) [reply AirCasting/SessionViews/ShareViewModal.swift:30]: Sorry for that, I've changed it to another. IDK why it left by itself from my commit

**Pull request merged**